packages/hooks/src/hooks/usePoller.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useRef } from "react";
+
+const usePoller = (func: () => void, delay: number): void => {
```

Comment:
We should follow the pattern for the other hooks and export this directly rather than using a `default` export.

```suggestion
export const usePoller = (func: () => void, delay: number): void => {
```